lib/rbovirt.rb

```diff
@@ -97,9 +97,12 @@ def floppy_hook?
     end
 
     private
+
     def search_url opts
-      search = opts[:search] || ("datacenter=%s" % current_datacenter.name)
-      "?search=%s" % CGI.escape(search)
+      search = opts[:search] || ("datacenter=\"%s\"" % current_datacenter.name)
+      search += " page #{opts[:page]}" if opts[:page]
+      max = opts[:max] ? ";max=#{opts[:max]}" : ''
```

Comment:
nitpick: you are not consistent in your style, a line above uses if, while this line doesn't :)